src/daemon/daemon.cpp

```diff
@@ -2662,11 +2663,26 @@ error_string mp::Daemon::async_wait_for_ssh_and_start_mounts_for(const std::stri
                 }
                 catch (const std::exception& e)
                 {
-                    fmt::format_to(errors, "Removing \"{}\": {}\n", target_path, e.what());
+                    // TODO: Combine these into one warning level log once they are displayed in the cli by
+                    // default
+                    mpl::log(mpl::Level::info, category, fmt::format("Removing \'{}\': {}\n", target_path, e.what()));
+                    fmt::format_to(warnings,
+                                   fmt::format("Removing mount \'{}\' from {}: {}\n", target_path, name, e.what()));
```

Comment:
i don't think single quotes need to be escaped like that, they should be printed just fine without the backslash